range.go

```diff
@@ -2,16 +2,27 @@ package protocol
 
 type Range struct {
 	Vertex
+	RangeData
+	Tag *RangeTag `json:"tag,omitempty"`
+}
+
+type RangeData struct {
 	Start Pos `json:"start"`
 	End   Pos `json:"end"`
 }
 
+type RangeTag struct {
```

Comment:
There's no comment on these types so I'm a bit confused here. It also looks like this is still a non-standard addition to the spec. Why does a range with a tag have two ranges? What is the type? Does it correspond to https://microsoft.github.io/language-server-protocol/specifications/lsif/0.5.0/specification/#documentSymbol?

Reply:
Yep, corresponds to the `DeclarationTag` in the LSIF spec: https://microsoft.github.io/language-server-protocol/specifications/lsif/0.5.0/specification/#documentSymbol. Added a docstring.